src/mimemail.erl

```diff
@@ -900,7 +946,7 @@ encode_quoted_printable(Body, Acc, L) when L >= 75 ->
 		SIndex when (L - 75) < SIndex ->
 			% okay, we can safely stick some whitespace in
 			NewAcc = insert_soft_newline(Acc, SIndex - 1),
-			encode_quoted_printable(Body, NewAcc, 0);
+			encode_quoted_printable(Body, NewAcc, SIndex);
 		_ ->
```

Comment:
this and next diff are actually bugfixes. Here we used to set "current line length" to 0 even though the line we are building might be much longer than 0.